homework/index.js

```diff
@@ -29,18 +14,94 @@
     });
     return elem;
   }
+  function createRepositoryDescription(repoContainer, repository) {
+    repoContainer.innerHTML = '';
+    const table = createAndAppend('table', repoContainer);
+    const tBody = createAndAppend('tbody', table);
+    const details = ['Repository:', 'Description:', 'Forks:', 'Updated:'];
+    details.forEach(detail => {
+      const tr = createAndAppend('tr', tBody);
+      createAndAppend('td', tr, { class: 'label', text: detail });
+      createAndAppend('td', tr, { id: detail });
+    });
+    const secondTd = document.getElementById('Repository:');
+    const link = createAndAppend('a', secondTd, {
+      href: repository.html_url,
+      target: '_blank',
+    });
+    link.innerText = repository.name;
```

Comment:
As the `createAndAppend` function arguments, you may specify the text while you are creating the element by it self.